selfdrive/car/torque_data/override.yaml

```diff
@@ -21,7 +21,7 @@ FORD FOCUS 4TH GEN: [.nan, 1.5, .nan]
 COMMA BODY: [.nan, 1000, .nan]
 
 # Totally new cars
-RAM 1500 5TH GEN: [2.0, 2.0, 0.0]
+RAM 1500 5TH GEN: [2.0, 2.5, 0.0]
```

Comment:
you'll also want to raise the factor -- the first variable. the second is just for documentation purposes